openslides/core/static/templates/index.html

```diff
@@ -3,109 +3,162 @@
+                </div>
+                <div class="modal-body">
+                  <p ng-if='loginFailed' class="text-danger">
+                    <strong translate>Username or password is not correct.</strong>
+                  <form>
+                    <div class="input-group form-group">
+                      <div class="input-group-addon"><i class="fa fa-user"></i></div>
+                      <input type="text" ng-model="username" class="form-control input-lg"
+                          placeholder="{{'Username'|translate}}">
+                    </div>
+                    <div class="input-group form-group">
+                      <div class="input-group-addon"><i class="fa fa-key"></i></div>
+                      <input type="password" ng-model="password" class="form-control input-lg"
+                          placeholder="{{'Password'|translate}}">
+                    </div>
+                    <div class="form-group">
+                      <button type="submit" ng-click="login(username, password)"
+                          class="btn btn-primary btn-lg btn-block" translate>
+                        Login
+                      </button>
                     </div>
-                    <div ng-if="!operator.isAuthenticated()">
-                        <a href="#" ng-click="showLoginForm = true" class="btn btn-default">
-                            <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
-                            {{ gettext("Login") }}
-                        </a>
-                        <div ng-if="showLoginForm">
-                            <form>
-                                username: <input type="text" ng-model="username"><br>
-                                password: <input type="password" ng-model="password"><br>
-                                <input type="submit" ng-click="login(username, password)" value="Save" />
-                            </form>
-                        </div>
+                    <div class="form-group">
+                      <!-- TODO: if anonymous user is activate -->
+                      <button type="submit" class="btn btn-default" translate>
+                        Continue as guest
+                      </button>
                     </div>
+                  </form>
                 </div>
+              </div>
             </div>
+          </div>
         </div>
-    </nav>
-    <!-- Container -->
-    <div class="container-fluid" id="container">
-    <div class="row">
+      </div>
+      <!-- language switcher -->
+      <div class="btn-group" ng-controller="LanguageCtrl">
+        <button class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+          <i class="fa fa-flag"></i>
+          <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu dropdown-menu-right" role="menu">
+          <li><a href="" ng-click="switchLanguage('en')">
```

Comment:
Hm. Then you have to add type="button" or role="button" or both for assistive technologies.